README.md

```diff
@@ -1,117 +1,200 @@
 <!-- markdownlint-disable MD004 -->
-# Overview
+
+# AKS middleware for Go
+
+<!-- vscode-markdown-toc -->
+* 1. [Usage](#Usage)
```

Comment:
Not sure if this is just an issue for me locally, but clicking the links does not take me to the section in the readme

Reply:
Fixed